#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_texfont_fontforge.dpatch by  <preining@debian.org>
##
## DP: texfont calls pfaedit, but Debian has already fontforge, change the name

@DPATCH@
--- ./scripts/context/perl/texfont.pl.orig	2006-12-09 12:19:30.000000000 +0100
+++ ./scripts/context/perl/texfont.pl	2006-12-09 12:20:01.000000000 +0100
@@ -818,8 +818,8 @@
         report ("destination file : $pfbfont") ; }
     else
       { error ("missing otf/ttf source file") }
-    open (CONVERT, "| pfaedit -script -") || error ("couldn't open pipe to pfaedit") ;
-    report ("pre-processing with : pfaedit") ;
+    open (CONVERT, "| fontforge -script -") || error ("couldn't open pipe to fontforge") ;
+    report ("pre-processing with : fontforge") ;
     print CONVERT "Open('$infont');\n Generate('$pfbfont', '', 1) ;\n" ;
     close (CONVERT) }
 
